paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/ExtendedPaymentElementConfirmationTestActivity.kt

```diff
@@ -123,6 +124,10 @@ internal interface ExtendedPaymentElementConfirmationTestModule {
     fun bindLinkGateFactory(linkGateFactory: DefaultLinkGate.Factory): LinkGate.Factory
 
     companion object {
+        @Provides
+        @PaymentElementCallbackIdentifier
+        fun providesInstanceId(): String = "ExtendedConfirmation"
```

Comment:
Should we name this something test specific, as is, it seems like it could easily get confused with production code.